src/windowsservercore/ltsc2019/helix/amd64/Dockerfile

```diff
@@ -1,29 +1,30 @@
 # escape=`
 FROM mcr.microsoft.com/windows/servercore:ltsc2019
 
-USER ContainerAdministrator
-
-SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
-
 # Install latest stable version of Python
-RUN $apiUrl = 'https://api.nuget.org/v3-flatcontainer/python/index.json'; `
-    $response = Invoke-RestMethod -Uri $apiUrl; `
-    $versions = $response.versions | Where-Object { $_ -notmatch '-' } | Sort-Object { [version]$_ } -Descending; `
-    $latestVersion = $versions[0]; `
-    Invoke-WebRequest -Uri https://www.nuget.org/api/v2/package/python/$latestVersion -OutFile $env:TEMP\python.zip; `
-    md C:\Python; `
-    md C:\PythonTemp; `
-    tar -zxf  $env:TEMP\python.zip -C C:\PythonTemp; `
-    xcopy /s c:\PythonTemp\tools C:\Python; `
-    Remove-Item -Recurse -Force C:\PythonTemp; `
-    Remove-Item -Force $env:TEMP\python.zip;"
+RUN powershell -Command "`
+        $ErrorActionPreference = 'Stop'; `
+        $ProgressPreference = 'SilentlyContinue'; `
+        $apiUrl = 'https://api.nuget.org/v3-flatcontainer/python/index.json'; `
+        $response = Invoke-RestMethod -Uri $apiUrl; `
+        $versions = $response.versions | Where-Object { $_ -notmatch '-' } | Sort-Object { [version]$_ } -Descending; `
+        $latestVersion = $versions[0]; `
+        echo \"Downloading Python $latestVersion\"; `
+        Invoke-WebRequest -Uri https://www.nuget.org/api/v2/package/python/$latestVersion -OutFile $env:TEMP\python.zip; `
+        md C:\Python; `
+        md C:\PythonTemp; `
+        tar -zxf  $env:TEMP\python.zip -C C:\PythonTemp; `
+        xcopy /s c:\PythonTemp\tools C:\Python; `
+        Remove-Item -Recurse -Force C:\PythonTemp; `
+        Remove-Item -Force $env:TEMP\python.zip;"
 
-SHELL ["cmd", "/S", "/C"]
+ENV PATH="$PATH;C:\Python;C:\python\scripts" `
```

Comment:
Does this work?  This used to never be supported.  IIRC you have to use setx to set the path in Windows Dockerfiles.

Reply:
Yes, it works. Otherwise, the build would fail due to python not being available from the PATH. 😄 But yes, I recall this being an issue in the past but more recent Docker versions seem to have resolved this.

Reply:
Oh, I see. There's an issue in being able to reference another variable from this instruction.

```Dockerfile
ENV PATH="$PATH;C:\Python;C:\python\scripts"
```

Gives you a `PATH` of `;C:\Python;C:\python\scripts`. If you change `$PATH` to `%PATH%`, you get a literal value of `%PATH%;C:\Python;C:\python\scripts`.

Using `setx` solves that.